src/test/java/io/github/syst3ms/skriptparser/parsing/SyntaxParserTest.java

```diff
@@ -71,9 +71,6 @@ public Iterator<DynamicNode> syntaxTest() {
                             var allErrors = new ArrayList<>(errorsFound);
                             Set<String> duplicateErrors = new HashSet<>();
                             allErrors.removeIf(val -> !duplicateErrors.add(val.getMessage()));
-
-                            // Reset variables
-                            Variables.clearVariables();
```

Comment:
All tests assume no user-stored variables are used. Did you check that all tests passed? Seems like some might not?

Reply:
I see, that did it.